libs/openFrameworks/3d/ofEasyCam.h

```diff
@@ -219,22 +222,21 @@ class ofEasyCam : public ofCamera {
 
     /// \brief The current rotation quaternion.
 	glm::quat curRot;
-
-    /// \brief The previous X axis.
-	glm::vec3 prevAxisX;
-
-    /// \brief The previous Y axis.
-	glm::vec3 prevAxisY;
-
-    /// \brief The previous Z axis.
-	glm::vec3 prevAxisZ;
-
-    /// \brief the previous camera position.
-	glm::vec3 prevPosition;
-
-    /// \brief The previous camera orientation.
-	glm::quat prevOrientation;
-
+	
+	/// \name On Press cache
+	/// \{
+	/// \brief camera properties when the mouse is pressed.
+
+	glm::vec3 onPressAxisX;
```

Comment:
More a cosmetic comment: `onSomething` might suggest that these variables have something to do with events - or callbacks, where this naming pattern is used frequently. If I understand this section correctly, these members are here to capture previous interactive camera state. 

In other parts of the openFrameworks codebase, we're using "previousSoundSo" or "lastSoundSo", and it might make things a bit more easy to read if consistent.